src/Components/RowCalcul.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ]  Please remove all the eslint disable statement. Because Eslint helps developers to write clean JavaScript codes and adapt best practices when writing the code. You just need to export it as the default code. smile